docker/pulsar-all/pom.xml

```diff
@@ -119,57 +130,71 @@
             </executions>
           </plugin>
           <plugin>
-            <groupId>com.spotify</groupId>
-            <artifactId>dockerfile-maven-plugin</artifactId>
-            <version>${dockerfile-maven.version}</version>
+            <groupId>io.fabric8</groupId>
+            <artifactId>docker-maven-plugin</artifactId>
             <executions>
               <execution>
                 <id>default</id>
+                <phase>package</phase>
                 <goals>
                   <goal>build</goal>
                 </goals>
-              </execution>
-              <execution>
-                <id>add-no-repo-and-version</id>
-                <goals>
-                  <goal>tag</goal>
-                </goals>
                 <configuration>
-                  <repository>pulsar-all</repository>
-                  <tag>${project.version}</tag>
+                  <images>
```

Comment:
By the way, I notice that there are redundant configurations here, and I think we can simplify them.

```xml
<executions>
      <execution>
        <id>default</id>
        <phase>package</phase>
        <goals>
          <goal>build</goal>
          <goal>push</goal>
        </goals>
        <configuration>
          <images>
            <image>
              <name>${docker.organization}/pulsar</name>
              <build>
                <contextDir>${project.basedir}</contextDir>
                <tags>
                  <tag>latest</tag>
                  <tag>${project.version}</tag>
                </tags>
              </build>
            </image>
          </images>
        </configuration>
      </execution>
</executions>
```

Then maybe we also need to update the https://github.com/apache/pulsar/blob/master/docker/publish.sh#L66

So like using `${docker.organization}/pulsar-all:latest` instead of `pulsar-all:latest`.

Reply:
See https://github.com/apache/pulsar/pull/17148#discussion_r949713557